Source/WebCore/platform/graphics/gstreamer/MediaPlayerPrivateGStreamer.cpp

```diff
@@ -213,6 +213,9 @@ MediaPlayerPrivateGStreamer::~MediaPlayerPrivateGStreamer()
         ASSERT(bus);
         gst_bus_set_sync_handler(bus.get(), nullptr, nullptr, nullptr);
         g_signal_handlers_disconnect_matched(m_pipeline.get(), G_SIGNAL_MATCH_DATA, 0, 0, nullptr, nullptr, this);
+        // BRCM specific, flush to unlock aud/vid filters
```

Comment:
WPE_BACKEND is not available in WebCore and WPE_BACKEND_BCM_NEXUS is defined only for WPE module, changing that seems to be out of current task scope.